temporal-sdk/src/main/java/io/temporal/internal/common/HistoryJsonUtils.java

```diff
@@ -83,26 +78,4 @@ private static String convertEnumValues(
     }
     return parsed.jsonString();
   }
-
```

Comment:
This code was generalized and moved to ProtobufEnumNameUtils to be reused in Search Attributes code